DependencyInjection/HttplugExtension.php

```diff
@@ -307,7 +370,7 @@ private function configureAutoDiscoveryClients(ContainerBuilder $container, arra
                     $container,
                     'auto_discovered_client',
                     [HttpClientDiscovery::class, 'find'],
-                    $config['profiling']['enabled']
+                    $this->isConfigEnabled($container, $config['profiling'])
```

Comment:
its the counterpart to canBeEnabled/canBeDisabled. i feel when we use canBeEnabled/Disabled we should also use this method.